orocos.autobuild

```diff
@@ -28,16 +29,17 @@ cmake_package 'rtt' do |pkg|
 
     # Disable building the test suite because it needs a lot of time and memory
     pkg.with_tests
+    pkg.test_utility.source_dir = File.join(pkg.srcdir, 'tests')
```

Comment:
Shouldn't that be "test" to align with the Rock.cmake macros?

Reply:
This is the source folder. We don't get to pick (this is `rtt`).